.github/workflows/pylint.yml

```
@@ -0,0 +1,23 @@
+name: Pylint
+
+on: [push]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)